README.md

```diff
@@ -28,6 +28,11 @@ df.head()
 
 ```
 
+To access data in BigQuery, users will need to have the credentials of a service account
+with "BigQuery Guest" and "BigQuery Dataset Guest" roles.  If using this library through
+a notebook derived from the datalab-jupyter image, the credentials should be stored in
+the repo root.  Otherwise, `EBMDATALAB_BQ_CREDENTIALS_PATH` should be set to the path of
```

Comment:
Personally, I always find working out how to provision a service account a nightmare. I just tried again for 10 minutes and am still not sure; so for less technical users this is likely to be a bit horrible.

So is there more we can do here to help? I'm also assuming we (Bennett) are the only users of this, so we have the option "ask tech support" (though probably we then need to add something to our team manual / playbooks about it).

For example:
 * can we explain what "the credentials should be stored in the repo root" means? i.e. in what format? 
 * do we expect users to work out how to make their own service account? 
 * do we expect BI users to share the same service account? Will it live in Bitwarden?